### PR TITLE
OGM-917 Supporting embeddable key with single column

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

**Comment by Sanne** [comment]
boooo booooo ! it's failing checkstyle:D

**Comment by gunnarmorling** [comment]
Wot?! Can't believe it ;)

2015-12-16 14:33 GMT+01:00 Sanne Grinovero <notifications@github.com>:

File: mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/mapping/EmbeddableIdMappingTest.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	@Before
+	public void setUpTestData() {
+		OgmSession session = openSession();
+		Transaction transaction = session.beginTransaction();
+
+		session.persist( new SingleBoardComputer( new SingleBoardComputerPk( "sbc-1" ), "Raspberry Pi" ) );
+
+		session.persist(
+				new News(
+						new NewsID( "seven-projects", "Jerry" ),
```
Comment (Sanne) [inline mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/mapping/EmbeddableIdMappingTest.java]: Why is this test which also uses a dual-property embedded key specific to MongoDB only?

Comment (gunnarmorling) [inline mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/mapping/EmbeddableIdMappingTest.java]: It's a test for the specific persistent mapping, here we are asserting the
MongoDB-specific JSON representation.

**Comment by Sanne** [comment]
merged after biting of some whitespace

**Pull request closed without merging**